plugin/src/main/kotlin/com/github/triplet/gradle/play/internal/Listings.kt

```diff
@@ -3,13 +3,16 @@ package com.github.triplet.gradle.play.internal
 import java.io.File
 import java.io.FileFilter
 
+// region '419' is a special case in the Play Store that represents latin america
+// 'fil' is a special case in the Play Store that represents Filipino
+private val localeRegex = Regex("^(fil|[a-z]{2}(-([A-Z]{2}|419))?)\\z")
```

Comment:
Can we move this and the `imageExtensions` inside their respective filter objects? Makes it clearer what they're used for.